walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,61 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
+
+  /**
+   * Returns collection of time slots, in which we can appoint the meeting.
+   * @param events   All events in the day.
+   * @param request  Information about event that we want to add (attendees, required duration).
+   * */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // hash map, where key is the time moment and object is the number of attendees from the request,
+    // who will be busy or free from that time moment
```

Comment:
Not very clear, explain that it is the overall change in the number of available attendees at that particular time.
Maybe a better name for the variable could help convey that better as well.